sdk/component/store/local/sqlite/store.go

```diff
@@ -76,12 +77,17 @@ func NewManager(ctx context.Context, opts ...managerOption) (*manager, error) {
 
 	// Create sqlite database file if not exists.
 	if _, err := os.Stat(mgr.dsn); err != nil {
-		if !os.IsNotExist(err) {
+		if os.IsNotExist(err) {
```

Comment:
Find a bug here when running multiple workflows locally with smithyctl and putting the db into a folder - this fixes the issue by creating nested folders, if needed.